src/main/java/cz/muni/fi/pa165/bluebat/dao/ExcursionDao.java

```diff
@@ -0,0 +1,18 @@
+package cz.muni.fi.pa165.bluebat.dao;
+
+import cz.muni.fi.pa165.bluebat.entity.Excursion;
+
+/**
+ * Created by Tomáš Hampl on 3.3.21.
+ */
```

Comment:
U interfaců to asi bude chtít trošku obsáhlejší Javadoc - je to i v popisu milestonu.